### DISCUSSION TITLE
How to enable themeing for React Navigation 6

### DISCUSSION THREAD

**Discussion by Amocon** [tamagui/tamagui#926] [Q&A]
I am new to Tamagui and build a simple app to test it a bit. Now I've run into a problem because I haven't found a way to change the style of the React Navigation header or footer to match the theme I've chosen in Tamagui. If you have a solution, I would be very happy to hear about it.

**Comment by byteab** [comment]
you can use `useTheme` hook from `tamagui` with combination to `react navigation` theme prop

```js
export default function App() {
  const theme = useTheme()

  // you can put theme inside `useMemo` as well
  const myTheme = {
      colors: {
          primary: theme.primary
      }
  }
  return (
    <NavigationContainer theme={myTheme}>{/* content */}</NavigationContainer>
  );
}
```
check these links
https://tamagui.dev/docs/core/use-theme
https://reactnavigation.org/docs/themes/
